src/rpcblockchain.cpp

```diff
@@ -2042,6 +2044,13 @@ Value execute(const Array& params, bool fHelp)
             entry.push_back(Pair("Requested a quorum - waiting for resolution.",1));
             results.push_back(entry);
     }
+    else if (sItem == "tally")
```

Comment:
I think we should drop this one even with the transition improvement. It messes with the SB load order.